XPlat.Devices.Geolocation/Geolocator.Android.cs

```diff
@@ -33,12 +33,9 @@ public class Geolocator : IGeolocator
         /// <summary>
         /// Initializes a new instance of the <see cref="Geolocator"/> class.
         /// </summary>
-        /// <param name="context">
-        /// The Android context.
-        /// </param>
-        public Geolocator(Context context)
+        public Geolocator()
```

Comment:
I think it would be preferred if the original constructors were kept for all these file changes made.

The reason being is that by removing this, it would be a breaking change in the API as current users of the library would no longer be able to compile their application.

I'd suggest providing a default constructor alongside the original which passes the Application's context through, for example in this case:

```csharp
public Geolocator() : this(Android.App.Application.Context)
{
}
```

This way, a developer working with the library will be able to choose the default which will use the application context, or they can provide their own.

Reply:
You're absolutely right, I will work on that 😃

Reply:
Done 😄